libs/ledger-live-common/src/families/near/logic.ts

```diff
@@ -178,7 +179,7 @@ export const canWithdraw = (
 export const getYoctoThreshold = (): BigNumber => {
   return new BigNumber(10)
     .pow(new BigNumber(utils.format.NEAR_NOMINATION_EXP - FRACTIONAL_DIGITS))
-    .minus("1");
+    .minus(YOCTO_THRESHOLD_VARIATION);
```

Comment:
Adjusting this as I saw at least one case where the staked value returned from the node was 2 yoctoNEAR less than what was actually staked - giving us a slightly larger buffer for any variation we might see.